great_expectations/profile/user_configurable_profiler.py

```diff
@@ -401,7 +423,8 @@ def _add_column_type_to_column_info(self, profile_dataset, column_name):
 
         return column_type
 
-    def _get_column_type(self, profile_dataset, column):
+    @staticmethod
+    def _get_column_type(profile_dataset, column):
```

Comment:
Do we know the types here? Can we type hint if possible?

Reply:
@cdkini Yes -- added (`str`) -- in next commit.